### PR TITLE
fix travis errors

### PR DESCRIPTION
**Pull request by lioramilbaum**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,16 +9,28 @@ python:
...
+  apt:
+    update: true
+
+cache:
+  apt: true`
+
 install:
-  - pip install -r test-requirements.txt
-  - docker build -t devopsloft/devopsloft --build-arg REQUIREMENTS=requirements-dev.txt docker/.
+  - pip install -r requirements-dev.txt
```
Comment (ShimiTaNaka) [inline .travis.yml]: If the application runs in the container, it's redundant to install pip on the host, the tests should run from inside the container, where the application is served

File: .travis.yml [hunk 2]
```diff
@@ -9,16 +9,28 @@ python:
...
-  - docker cp tests web:/tests/
-  - docker exec -it web bash -c 'pip install pyyaml'
+  - docker run -d -p 80:80 -p 3306:3306 --name web devopsloft/devopsloft
+  - docker exec -it web bash -c 'while ! mysqladmin ping --silent; do sleep 3; done'
 
 script:
-    - flake8 --exclude venv .
-    - docker exec -it web bash -c 'python3 /tests/is_table_empty.py'
-    - docker exec -it web bash -c 'python3 /tests/basic_http_get_test.py'
+  - flake8 --exclude venv .
```
Comment (ShimiTaNaka) [inline .travis.yml]: See above, the tests should run in the scope of the container.
They can possibly run as a side container, but that depends on implementation

File: Vagrantfile [hunk 3]
```diff
@@ -63,8 +63,9 @@ Vagrant.configure("2") do |config|
   config.vm.provision "docker" do |d|
     d.build_image "/vagrant/docker",
       args: "-t devopsloft/devopsloft"
-    d.run "devopsloft/devopsloft",
-      args: "-p 80:80"
+    d.run "web",
```
Comment (ShimiTaNaka) [inline Vagrantfile]: I'm not 100% sure thy vagrant is building the image
isn't the image run *inside* the VM?

Comment (lioramilbaum) [reply Vagrantfile]: the image is built by Vagrant and than the container based on the image is spinned by Vagrant inside the VM

File: .travis.yml (hunk 1 above)
Comment (lioramilbaum) [reply .travis.yml]: Running tests inside the web container will require to install additional packages which might impact on the execution of the tested services.

File: .travis.yml (hunk 2 above)
Comment (lioramilbaum) [reply .travis.yml]: Spinning another container for tests is a good idea for implementation

File: .travis.yml (hunk 1 above)
Comment (lioramilbaum) [reply .travis.yml]: the tests should have zero foot print in the web container

**Pull request merged**